packages/recommender/package-lock.json

```diff
@@ -1,5326 +1,8 @@
 {
 	"name": "loopback4-example-recommender",
 	"version": "1.1.1",
-	"lockfileVersion": 2,
```

Comment:
@mrmodise is it intended that the content of all the package-lock files is removed?

Reply:
not intended this file had conflicts earlier

Reply:
@nflaig it looks like version 1 is the correct version. It gets changed when I install. See warning from pipeline and locally on this:

`npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
`